### PR TITLE
{compiler}[dummy] GCC 5.5.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2441.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/630c4c8997e915020eaf190e283e9b51 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2125.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2cdf92c17b2c707c780c7b5fc9c419bf for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
cermis - Linux debian 8.8, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/3b07191fbccd0cc07915e572f119a64e for a full test report.

File: easybuild/easyconfigs/b/Bison/Bison-3.0.4-GCCcore-5.5.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+homepage = 'http://www.gnu.org/software/bison'
+description = """Bison is a general-purpose parser generator that converts an annotated context-free grammar
+ into a deterministic LR or generalized LR (GLR) parser employing LALR(1) parser tables."""
+
+toolchain = {'name': 'GCCcore', 'version': '5.5.0'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/b/Bison/Bison-3.0.4-GCCcore-5.5.0.eb]: Missing checksum

File: easybuild/easyconfigs/b/binutils/binutils-2.26-GCCcore-5.5.0.eb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+version = '2.26'
+
+homepage = 'http://directory.fsf.org/project/binutils/'
+description = "binutils: GNU binary utilities"
+
+toolchain = {'name': 'GCCcore', 'version': '5.5.0'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/b/binutils/binutils-2.26-GCCcore-5.5.0.eb]: Missing checksum

File: easybuild/easyconfigs/h/help2man/help2man-1.47.5-GCCcore-5.5.0.eb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+version = '1.47.5'
+
+homepage = 'https://www.gnu.org/software/help2man/'
+description = """help2man produces simple manual pages from the '--help' and '--version' output of other commands."""
+
+toolchain = {'name': 'GCCcore', 'version': '5.5.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_XZ]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/h/help2man/help2man-1.47.5-GCCcore-5.5.0.eb]: Missing checksum

File: easybuild/easyconfigs/m/M4/M4-1.4.18-GCCcore-5.5.0.eb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+homepage = 'http://www.gnu.org/software/m4/m4.html'
+description = """GNU M4 is an implementation of the traditional Unix macro processor. It is mostly SVR4 compatible
+  although it has some extensions (for example, handling more than 9 positional parameters to macros).
+ GNU M4 also has built-in functions for including files, running shell commands, doing arithmetic, etc."""
+
+toolchain = {'name': 'GCCcore', 'version': '5.5.0'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/m/M4/M4-1.4.18-GCCcore-5.5.0.eb]: Missing checksum

File: easybuild/easyconfigs/z/zlib/zlib-1.2.11-GCCcore-5.5.0.eb [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+description = """zlib is designed to be a free, general-purpose, legally unencumbered -- that is,
+ not covered by any patents -- lossless data-compression library for use on virtually any
+ computer hardware and operating system."""
+
+toolchain = {'name': 'GCCcore', 'version': '5.5.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://zlib.net/fossils']
+sources = [SOURCELOWER_TAR_GZ]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/z/zlib/zlib-1.2.11-GCCcore-5.5.0.eb]: Missing checksum

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2514.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/bdc3575effcfc4f568a0e947db47b21f for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**